package.json

```diff
@@ -64,5 +66,13 @@
     "plugins": [
       "transform-object-assign"
     ]
+  },
+  "config": {
+    "ghooks": {
+      "commit-msg": "validate-commit-msg"
+    },
+    "validate-commit-msg": {
+      "helpMessage": "Invalid commit message %s. Please see https://github.com/conventional-changelog/conventional-changelog/blob/v0.5.3/conventions/angular.md for information on how to format commit messages. Also, if this commit is on a development / WIP branch, you can disable this by running `git commit --no-verify`."
```

Comment:
> I would suggest adding a minimal sample

Great idea, will add. I don't really mind the length as it probably won't be changed that often.

> is there a description somewhere of what each of the types means?

https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#type